### PR TITLE
Feature/uc choosing

### PR DESCRIPTION
**Pull request by kiko-g**

### CODE REVIEW

**Review by TiagooGomess — CHANGES_REQUESTED** [review]
Nice work. Just a suggestion: maybe we don't need the autocomplete for fields that are numbers, `Semestre` and `Ano Letivo`. There is an error in the console because of using numbers in the autocomplete:
Screenshot from 2021-12-15 19-08-27
Maybe we can have just a simple dropdown.

**Comment by TiagooGomess** [comment]
Are we going to use a 2 space indentation?

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": true
```
Comment (TiagooGomess) [inline .prettierrc:5]: Just a suggestion: 
```suggestion
    "tabWidth": 4,
    "useTabs": false,
    "semi": true,
    "singleQuote": false
```

**Review by TiagooGomess — CHANGES_REQUESTED** [review]

**Comment by TiagooGomess** [comment]
Please update the changed files tab width to the new configuration (4 instead of 2).

**Comment by kiko-g** [comment]
image

If everyone can agree on this configuration this is ready to merge

**Review by TiagooGomess — APPROVED** [review]

**Pull request merged**